srcpkgs/gimx/template

```diff
@@ -0,0 +1,80 @@
+# Template file for 'gimx'
+pkgname=gimx
+version=8.0
+revision=1
+build_style=gnu-makefile
+make_use_env=yes
+hostmakedepends="gettext pkg-config"
+makedepends="libbluetooth-devel libcurl-devel libusb-devel libX11-devel libXi-devel
+ libxml2-devel mhash-devel ncurses-devel wxWidgets-gtk3-devel"
+depends="gimx-cli-${version}_${revision} gimx-firmware-${version}_${revision}
+ avrdude xdg-utils xterm"
+short_desc="Use a computer as a hub for your gaming devices"
```

Comment:
This isn't the best desc, I can't really tell what it does.

Reply:
I agree. I just took the description from the homepage. I will come up with some more descriptive.

Reply:
I changed the description but it was too specific. It's difficult to describe what this software does in 71 characters or less. I reverted to the upstream description for now.